### PR TITLE
Changing Cloudflow operator to use alpine base image

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
### What changes were proposed in this pull request?
changing base image from ubuntu to alpine

### Why are the changes needed?
To avoid High Risk vulnerabilities found through Jfrog/Xray:

dpkg-source in dpkg 1.3.0 through 1.18.23 is able to use a non-GNU patch program and does not offer a protection mechanism for blank-indented diff hunks, which allows remote attackers to conduct directory traversal attacks via a crafted Debian source package, as demonstrated by use of dpkg-source on NetBSD.

In util-linux before 2.32-rc1, bash-completion/umount allows local users to gain privileges by embedding shell commands in a mountpoint name, which is mishandled during a umount command (within Bash) by a different user, as demonstrated by logging in as root and entering umount followed by a tab character for autocompletion.

Some HTTP/2 implementations are vulnerable to resource loops, potentially leading to a denial of service. The attacker creates multiple request streams and continually shuffles the priority of the streams in a way that causes substantial churn to the priority tree. This can consume excess CPU.
...

### Does this PR introduce any user-facing change?
No

### How was this patch tested?
Deploying in GKE the operator and deploying example app [call-record-aggregator](https://github.com/lightbend/cloudflow/tree/master/examples/call-record-aggregator)

### CODE REVIEW

File: core/build.sbt [hunk 1]
```diff
@@ -431,9 +430,11 @@ lazy val operator =
         val targetDir    = "/app"
 
         new Dockerfile {
-          from("adoptopenjdk/openjdk8:latest")
+          from("adoptopenjdk/openjdk8:alpine")
```
Comment (yuchaoran2011) [inline core/build.sbt:433]: Is there a version of Alpine we can specify here?

Comment (franciscolopezsancho) [reply core/build.sbt:433]: AFAIK that adoptopenjdk/openjdk8:alpine points to a 3.12

https://github.com/AdoptOpenJDK/openjdk-docker/blob/master/8/jdk/alpine/Dockerfile.hotspot.releases.full

Not sure we can add anything else @yuchaoran2011

Comment (yuchaoran2011) [reply core/build.sbt:433]: You verified that Alpine 3.9 didn't have critical vulnerabilities. Is that the case for 3.12 too?

Comment (franciscolopezsancho) [reply core/build.sbt:433]: Yep, I run the scan against this very image: adoptopenjdk/openjdk8:alpine

**Pull request merged**